app/models/ama.rb

```diff
@@ -31,6 +31,8 @@ class AMA < ApplicationRecord
   has_many :ama_subscribers, dependent: :destroy
 
   scope :for_original_post, ->(post) { where(original_post: post) }
+  scope :past_ama, -> { where('end_date <= ?', Date.today) }
+  scope :future_ama, -> { where('end_date > ?', Date.today) }
```

Comment:
Could we maybe use `>=` to here instead? I realize the logic is weird, but I think we want to be able to display an ama that happened today on the main page as well. That's the way Product Hunt does it at least.